src/shrutiC/MathematicalOperation.java

```diff
@@ -0,0 +1,63 @@
+package shrutiC;
+
+class MathematicalOperation {
+
+	void printEvenNos(int startIndex, int endIndex) {
+
+		System.out.println("\nEven numbers between " + startIndex + " and " + endIndex + " are :-");
+		int i = startIndex;
+		while (i <= endIndex) {
+			System.out.println(i);
```

Comment:
why we need to i++ twice.
if i give range from 11 then your code will not work.

& You can handle this scenario without creating third variable, think on this.
can you use startNumber++ for increment?